app/controllers/wpcc/your_results_controller.rb

```diff
@@ -53,5 +56,13 @@ def salary_message
         salary_frequency: session[:salary_frequency]
       )
     end
+
+    def default_salary_frequency
+      Hash[
+        Wpcc::YourDetailsForm::SALARY_FREQUENCIES.map do |elem|
+          [elem, elem == 'year' ? 'month' : elem]
```

Comment:
I think the issue is more that when we return to this code in 6 months time, we will wonder why the h we have changed year to month. The pr description and salary_frequency_spec and a commit all include the business reason for doing so. I think the Constant is fine as it is using accepted ruby expressions.